api/src/main/resources/workbench-api.yaml

```diff
@@ -4863,7 +4863,7 @@ definitions:
       enableRStudioGKEApp:
         type: boolean
         default: false
-        description: Whether users are able to use the RStudio GKE app.
+        description: Whether users are able to use the RStudio GKE app./criteria
```

Comment:
Is this a typo?

Reply:
doh... was thinking I was searching for /criteria :-). Fixed it. Thanks